python/cudf/cudf/tests/test_binops.py

```diff
@@ -377,7 +377,7 @@ def test_reflected_ops_scalar(func, dtype, obj_class):
     ps_result = func(random_series)
```

Comment:
This test is supposed to test whether the result of `func` is similar between cudf and pandas/numpy. `gen_rand` returned a numpy series which we construct a cudf series from, and then called `func` on it. If `gen_rand_series` returns a `cudf.Series` then this line will again call the cudf version of `func` instead of the  pandas/numpy version and we won't be testing anything.